webpack/scenes/ContentViews/Details/Filters/MatchContentModal/CVRpmMatchContentModal.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useCallback } from 'react';
+import { useSelector, shallowEqual } from 'react-redux';
+import { STATUS } from 'foremanReact/constants';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  Modal, ModalVariant,
+} from '@patternfly/react-core';
+import { TableVariant } from '@patternfly/react-table';
+import useDeepCompareEffect from 'use-deep-compare-effect';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import TableWrapper from '../../../../../components/Table/TableWrapper';
+import { getRPMMatchingContent } from '../../ContentViewDetailActions';
+import { selectRPMMatchContent, selectRPMMatchContentStatus } from '../../ContentViewDetailSelectors';
+
+
+const columnHeaders = [
+  __('RPM name'),
+  __('Summary'),
+];
+
+const emptyContentTitle = __('No matching RPM found.');
+const emptyContentBody = __("Given criteria doesn't match any RPMs. Try changing your rule.");
+const emptySearchTitle = __('Your search returned no matching RPMs.');
+const emptySearchBody = __('Try changing your search criteria.');
+
+const CVRpmMatchContentModal = ({ filterId, onClose, filterRuleId }) => {
+  const [searchQuery, updateSearchQuery] = useState('');
+  const [rows, setRows] = useState([]);
+  const [metadata, setMetadata] = useState({});
+  const response = useSelector(state => selectRPMMatchContent(state), shallowEqual);
+  const status = useSelector(state => selectRPMMatchContentStatus(state), shallowEqual);
+  const loading = status === STATUS.PENDING;
+
+  const fetchItems = useCallback(params =>
+    getRPMMatchingContent({
+      content_view_filter_rule_id: filterRuleId, filterId, ...params,
+    }), [filterRuleId, filterId]);
+
+  useDeepCompareEffect(() => {
+    const { results, ...meta } = response;
+    setMetadata(meta);
+
+    if (!loading && results) {
+      const newRows = [];
+      results.forEach(({
+        nvra, summary = '-', id, ...rest
+      }) => {
+        newRows.push({
+          cells: [
+            { title: <a rel="noreferrer" target="_blank" href={urlBuilder(`packages/${id}`, '')}>{nvra}</a> },
+            { title: summary },
+          ],
+          ...rest,
+        });
+      });
+
+      setRows(newRows);
+    }
+  }, [response, loading]);
+
+  return (
+    <Modal
+      title={__('Matching content')}
+      variant={ModalVariant.medium}
+      isOpen
+      onClose={onClose}
+      appendTo={document.body}
+    >
+      <TableWrapper
```

Comment:
Now that it's available, why not use the composable TableWrapper :angel:

Reply:
https://github.com/Katello/katello/pull/9550

Reply:
Le sigh! :stuck_out_tongue_closed_eyes: